DataManagementSystem/Utilities/CatalogUtilities.py

```diff
@@ -0,0 +1,138 @@
+########################################################################
+# $HeadURL$
+########################################################################
+
+""" DIRAC FileCatalog utilities
```

Comment:
The check done for the catalogs is not the same as for the Storage classes. Now, there is another utility in /Resources/Storage/Utilities